### PR TITLE
Create Sunaina

### PR DESCRIPTION
**Pull request by SunainaVijay**
My new file!!

### CODE REVIEW

**Comment by mothur-westcott** [comment]
I like your new file

File: Sunaina [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## I love
+
+- [ ] Music
+- [ ] Movies
```
Comment (chrisbSH) [inline Sunaina:4]: My favorite movie is What About Bob!

File: Sunaina [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## I love
+
+- [ ] Music
```
Comment (mothur-westcott) [inline Sunaina:3]: What kind of music?

Comment (cchay) [inline Sunaina:3]: That was my question.  What kind of music?

**Comment by SunainaVijay** [comment]
indian classical ... Rock .. Pop.. Everything that's ear catching!

**Pull request closed without merging**